daml-script/test/src/test-utils/com/daml/lf/engine/script/test/MultiParticipantFixture.scala

```diff
@@ -73,25 +82,41 @@ trait MultiParticipantFixture
       allowExistingSchema = false
     ),
   )
-  override protected lazy val suiteResource = {
+  override protected lazy val suiteResource: OwnedResource[ResourceContext, (Port, Port)] = {
     implicit val resourceContext: ResourceContext = ResourceContext(system.dispatcher)
     new OwnedResource[ResourceContext, (Port, Port)](
       for {
         _ <- Owner(
           kvutils.Config
             .createDefault(())
             .copy(
-              participants = Seq(participant1, participant2),
-              archiveFiles = Seq(darFile),
+              participants = Seq(participant1, participant2)
+            )
+        )
+        participant1Port = readPortfile(participant1Portfile)
+        participant2Port = readPortfile(participant2Portfile)
+        channel = {
```

Comment:
Do we clean this up after uploading packages?

Reply:
Oh, good point, thanks!

Reply:
Fixed.